rule-002/src/services/arango-client.service.ts

```diff
@@ -0,0 +1,39 @@
+import { Database } from 'arangojs';
+import { configuration } from '../config';
+import { LoggerService } from './logger.service';
+
+export class ArangoDBService {
+  client: Database;
+
+  constructor() {
+    this.client = new Database({
+      url: configuration.dbURL,
+      databaseName: configuration.dbName,
+      auth: {
+        username: configuration.dbUser,
+        password: configuration.dbPassword,
+      },
+    });
+
+    LoggerService.log('✅ ArangoDB connection is ready');
+  }
+
+  //   async query(query: string): Promise<string[][] | undefined | any> {
+  async query(query: string): Promise<any> {
+    try {
+      const cycles = await this.client.query(query);
+
+      const results = await cycles.batches.all();
+
+      LoggerService.log(`Query result: ${JSON.stringify(results)}`);
```

Comment:
Are we going to start logging content? During PoC @johanfol mentioned the reduction of speed by using console methods.